src/book/02-system/04-other/03-nugan.mdx

```diff
@@ -0,0 +1,193 @@
+
+In Windows, use the batch file rather than the environment file.
+
+```sh
+"./scripts/conda_deps.bat"
+```
+
+On non-Windows OS you could also use the provided shell script to install dependencies in Anaconda.
+
+```sh
+sh scripts/conda_deps.sh
+```
+
+### Training
+
+You will need to add a dataset. A semi-synthetic to real dataset can be found on the NAS.
+
+The dataset folder must contain three folders for training. These are
+
+| Folder     | Contents                                                               |
+| ---------- | ---------------------------------------------------------------------- |
+| trainA     | Contains training images in the Blender semi-synthetic style.          |
+| trainA_seg | Contains the corresponding segmentation images for the Blender images. |
+| trainB     | Contains training images in the real style.                            |
+
+Note that semi-synthetic images and segmentation images should have matched naming (such as the same number on both images in a pair). This ensures they pair correctly when sorted.
+
+Start the Visdom server to see a visual representation of the training.
+
+```sh
+python -m visdom.server
+```
+
+Open Visdom in your browser at [http://localhost:8097/](http://localhost:8097/).
+
+Open a new window. In Windows, run
+
+```sh
+"scripts/train.bat" <GPU_ID> <BATCH_SIZE> <NAME> <DATAROOT> <CONT> <EPOCHCOUNT>
+```
+
+In other OS, such as Linux, run
+
+```sh
+sh scripts/train.sh <GPU_ID> <BATCH_SIZE> <NAME> <DATAROOT> <CONT> <EPOCHCOUNT>
+```
+
+The options are as follows
+
+| Option     | Description                                                                                                                                    | Default                                    |
+| ---------- | ---------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------ |
+| GPU_ID     | ID of your GPU. For CPU, use -1.                                                                                                               | 0                                          |
+| BATCH_SIZE | Batch size for training. Many GPUs cannot support a batch size more than 1.                                                                    | 1                                          |
+| NAME       | Name of the training run. This will be the directory name of the checkpoints and results.                                                      | The current date prepended with 'soccer\_' |
+| DATAROOT   | The path to the folder with the trainA, trainA_seg, trainB folders.                                                                            | ./datasets/soccer                          |
+| CONT       | Set if you would like to continue training from an existing trained model. This does not include the base model, which is loaded if CONT is 0. | 0                                          |
```

Comment:
how do I select the file to load the weights or network if I do want to continue training on a previous run?

Reply:
Ah, that corresponds to using the same NAME as the previous run (if I understand your question correctly). I will clarify that.